engine/crates/parser-postgresql/src/registry/queries/delete_one.rs

```diff
@@ -0,0 +1,35 @@
+use common_types::auth::Operations;
+use engine::registry::{
+    resolvers::{
+        postgresql::{Operation, PostgresResolver},
+        Resolver,
+    },
+    MetaField, MetaInputValue,
+};
+use inflector::Inflector;
+use postgresql_types::database_definition::TableWalker;
+
+use crate::registry::context::{InputContext, OutputContext};
+
+pub(crate) fn register(
+    input_ctx: &InputContext<'_>,
+    table: TableWalker<'_>,
+    filter_oneof_type: &str,
+    output_ctx: &mut OutputContext,
+) {
+    let type_name = input_ctx.delete_type_name(table.client_name());
+    let query_name = format!("{}_Delete", table.client_name()).to_camel_case();
```

Comment:
Eventually the inserts will be able to return nested objects. This will happen in an additional select, and this will be possible when we use the TCP connection method together with transactions.